build/sync/plan/checks_test.go

```diff
@@ -8,8 +8,8 @@ import (
 	"path/filepath"
 	"testing"
 
+	git "github.com/go-git/go-git/v5"
```

Comment:
Does `build/go.mod` need to get updated?

Reply:
I've updated go.mod.